example/build.gradle

```diff
@@ -30,11 +30,6 @@ android {
     }
 
     buildTypes {
-        debug {
-            minifyEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android.txt')
-        }
```

Comment:
Good question and sorry for removing without an explanation. When I saw it I assumed it was there by accident, since it's extremely unusual to run ProGuard on the *debug* variant of a *sample* project. Why exactly is it needed?